smartsim/_core/_install/builder.py

```diff
@@ -395,12 +397,32 @@ def build_from_git(self, git_url, branch, device):
             "clone",
             "--recursive",
             git_url,
-            "--branch",
-            branch,
-            "--depth=1",
-            "RedisAI",
         ]
+        # Circumvent a bad `get_deps.sh` script from RAI on 1.2.7 with ONNX
+        # TODO: Look for a better way to do this or wait for RAI patch
```

Comment:
Do we have a ticket open against this workaround?

Reply:
We do now!! https://github.com/CrayLabs/Poseidon/issues/396